doc/docs.md

```diff
@@ -4635,23 +4662,26 @@ if select {
 
 For special purposes there are some builtin fields and methods:
 
+```v
+ch := chan int{cap: 2}
+println(ch.try_push(42)) // `.success` if pushed, `.not_ready` if full, `.closed` if closed
+println(ch.len) // Number of items in the buffer
+println(ch.cap) // Buffer capacity
+println(ch.closed) // Whether the channel is closed
+```
+
 ```v
 struct Abc {
 	x int
 }
 
-a := 2.13
-ch := chan f64{}
-res := ch.try_push(a) // try to perform `ch <- a`
-println(res)
-l := ch.len // number of elements in queue
-c := ch.cap // maximum queue length
-is_closed := ch.closed // bool flag - has `ch` been closed
-println(l)
-println(c)
-mut b := Abc{}
-ch2 := chan Abc{}
-res2 := ch2.try_pop(mut b) // try to perform `b = <-ch2`
```

Comment:
These were more informative compared to a whole example for just `.try_pop()`, with a comment that duplicates the code ...
Remove the new example, and restore the old code.

Reply:
Both have the same amount of information.🤔 It is the same information, however, without the use of variables.

Reply:
No, it is not the same information at all. The code sample before showed several channel features, while later in your example, shows just this, that is related to the channel use:
```v
res := ch.try_pop(mut b) // try to perform `b = <-ch`
```

Do delete the new code, and restore the old in this particular case, I do not want to argue about it.

Reply:
The other examples are mostly fine, but this one loses information, that is important for understanding how to use channels.